app/src/test/java/com/example/concordia_campus_guide/HelpersTest/IndoorPathHeuristicTest.java

```diff
@@ -45,71 +48,75 @@
+                entranceHBuildingWalkingPoint, elevatorWalkingPointH9,entranceMBBuildingWalkingPoint, walkingPoint4, classRoomInMBBuildingWalkingPoint);
 
-        when(mockWalkingPointDao.getAllAccessPointsOnFloor(walkingPoint1.getFloorCode(),PointType.ELEVATOR)).thenReturn(walkingPointList);
+        when(mockAppDb.walkingPointDao()).thenReturn(mockWalkingPointDao);
+        when(mockWalkingPointDao.getAllAccessPointsOnFloor(classRoomInHBuildingWalkingPoint1.getFloorCode(),PointType.ENTRANCE)).thenReturn(Arrays.asList(classRoomInHBuildingWalkingPoint1));
         when(mockWalkingPointDao.getAll()).thenReturn(walkingPointList);
         indoorPathHeuristic = new IndoorPathHeuristic(mockAppDb);
     }
 
+    private void settingUpWalkingPoints(){
+        classRoomInHBuildingWalkingPoint1 = new WalkingPoint(1,new Coordinates(1, 1), "H-9",new ArrayList<>(Arrays.asList(new Integer[]{2,3,4})), PointType.CLASSROOM, "937");
+        walkingPoint1 = new WalkingPoint(2,new Coordinates(2,2), "H-9", new ArrayList<>(Arrays.asList(new Integer[]{1,3,7})),PointType.NONE,"NONE");
+        walkingPoint2 = new WalkingPoint(3,new Coordinates(2,-2.5), "H-9", new ArrayList<>(Arrays.asList(new Integer[]{2,4,5})),PointType.NONE,"NONE");
+        walkingPoint3 = new WalkingPoint(4,new Coordinates(1, 1.5), "H-9", new ArrayList<>(Arrays.asList(new Integer[]{1,3,5,6})),PointType.NONE,"NONE");
+        classRoomInHBuildingWalkingPoint2 = new WalkingPoint(5,new Coordinates(5, 5), "H-9", new ArrayList<>(Arrays.asList(new Integer[]{3,4})),PointType.CLASSROOM,"921");
+        entranceHBuildingWalkingPoint = new WalkingPoint(6,new Coordinates(1.2, 2), "H-9", new ArrayList<>(Arrays.asList(new Integer[]{4,8})),PointType.ENTRANCE,"ENTRANCE");
+        elevatorWalkingPointH9 = new WalkingPoint(7,new Coordinates(1, 3), "H-9", new ArrayList<>(Arrays.asList(new Integer[]{2,11})),PointType.ELEVATOR,"elevators-1");
+        elevatorWalkingPointH8 = new WalkingPoint(11,new Coordinates(1, 3), "H-8", new ArrayList<>(Arrays.asList(new Integer[]{7,12})),PointType.ELEVATOR,"elevators-1");
+        entranceMBBuildingWalkingPoint = new WalkingPoint(8, new Coordinates(2.4,3.2),"MB-S2",new ArrayList<>(Arrays.asList(new Integer[]{7,9})),PointType.ENTRANCE, "ENTRANCE");
+        walkingPoint4 = new WalkingPoint(9,new Coordinates(-2.4,2.2),"MB-S2",new ArrayList<>(Arrays.asList(new Integer[]{8,10})),PointType.NONE, "NONE");
+        classRoomInMBBuildingWalkingPoint = new WalkingPoint(10,new Coordinates(2.45,-2.2),"MB-S2",new ArrayList<>(Arrays.asList(new Integer[]{9})),PointType.CLASSROOM, "279");
+        walkingPoint5 = new WalkingPoint(12,new Coordinates(2, 3), "H-8", new ArrayList<>(Arrays.asList(new Integer[]{11,13})),PointType.NONE,"NONE");
+        classRoomInHBuildingWalkingPointh8 =new WalkingPoint(13,new Coordinates(1, 1), "H-8",new ArrayList<>(Arrays.asList(new Integer[]{12})), PointType.CLASSROOM, "837");
+    }
+
     @Test
     public void getEuclideanDistanceTest(){
```

Comment:
Could you add a comment at the top of this class to indicate that the approach you took was that of characterization testing? Will make it easier to understand this file